messenger.rst

```diff
@@ -633,6 +634,132 @@ config and start your workers:
 
 See the `Supervisor docs`_ for more details.
 
+If a worker dependency like your database server is down, or timeout is reached,
+you can try to add reconnect logic, or just quit the worker (catch Exception,
+PDOException or do something more fine grained):
```

Comment:
At the end of this line we need to add two colons `::` instead of `:`  (that's the "magic syntax" that makes the following code example be highlighted as PHP).

Reply:
okay I did not know that